### DISCUSSION TITLE
5f9aa500-2950-4da0-8774-572e95fb30fd

### DISCUSSION THREAD

**Discussion by giscus** [ChooSeoyeon/blog-comment#24] [General]
# 5f9aa500-2950-4da0-8774-572e95fb30fd

객체지향적으로 설계하는 법을 모르겠다 하시는 분을 위한 ❗쉽게 따라해볼만한❗ 꿀꿀팁

https://choo.oopy.io/5f9aa500-2950-4da0-8774-572e95fb30fd

**Comment by CodingMasterLSW** [comment]
책 보면서 작년도 우테코 문제로 공부중인데 머리로는 알겠지만 감이 안 잡혔는데,  글 따라해보면서 야구게임에 적용해보니 감이 잡히는거 같아요. 정말 감사합니다!
